test/loader/DictionaryLoaderTest.js

```diff
@@ -48,3 +48,21 @@ describe("DictionaryLoader", function () {
         expect(dictionaries.token_info_dictionary.getFeatures("0")).to.have.length.above(1);
     });
 });
+
+describe("DictionaryLoader about loading", function () {
+    it("could load directory path without suffix /", function (done) {
+        var loader = DictionaryLoader.getLoader("dist/dict");// not have suffix /
```

Comment:
use case:

- `__dirname` have not suffix `/`.
- The result of `path.join` have not suffix `/`.

```js
var path = require("path");
var dictDir = path.join(__dirname, "dict");// => /path/to/current/dict
DictionaryLoader.getLoader(dictDir);
```